content/about.html

```diff
@@ -44,85 +43,37 @@ <h1 class="masthead-left__title no-link">
     </header>
 
     <main>
-      <div class="vision">
+      <div class="principles">
```

Comment:
We may need to keep this class name.

Reply:
Ah I bet this was done to make the text left align instead of center align. If that's the case then it's probably ok.

Reply:
I think this is because I removed the entire "vision" section, so the diff is a little screwy. I didn't consciously try to change any of the text alignment.

Reply:
oh ok. Even better! I never liked that centered text section :D